### PR TITLE
Another translation fix

### PR DESCRIPTION
**Pull request by boutinb**

### CODE REVIEW

File: JASP-Engine/JASP/po/R-nl.po [hunk 1]
```diff
@@ -1800,7 +1800,7 @@ msgstr ""
 "waarden bevatten, zijn niet meegenomen in de analyse."
 
 msgid "Descriptive Statistics"
-msgstr "Kengetallen"
+msgstr "Beschrijvende statistiek"
```
Comment (TimKDJ) [inline JASP-Engine/JASP/po/R-nl.po]: Statistiek

File: Resources/Help/analyses/anova_nl.md [hunk 2]
```diff
@@ -151,7 +151,7 @@ Q-Q Grafiek:
 - Met een Q-Q grafiek kan de normaliteit van de residuen visueel worden geïnspecteerd. De theoretische kwantielen staan op de x-as en de gestandaardiseerde residuen op de y-as. Hoe meer de punten op de diagonale lijn staan, hoe meer de data normaal verdeeld is. 
 
 #### Contrasten
-Afwijking/Enkelvoudig/Verschil/Helmert/Repeated/Polynoom/Custom Contrast:
+Afwijking/Enkelvoudig/Verschil/Helmert/Herhaald/Polynoom/Op maat Contrast:
```
Comment (TimKDJ) [inline Resources/Help/analyses/anova_nl.md]: Handmatig*

File: Resources/Help/analyses/anova_nl.md [hunk 3]
```diff
@@ -151,7 +151,7 @@ Q-Q Grafiek:
...
 - Vergelijking: De niveaus van de onafhankelijke variabele die worden vergeleken. Voor de custom contrasten wordt het gewicht getoond voor elk niveau van de factor.
```
Comment (TimKDJ) [inline Resources/Help/analyses/anova_nl.md]: custom = handmatige

File: Resources/Help/analyses/anovarepeatedmeasures_nl.md [hunk 4]
```diff
@@ -139,7 +139,7 @@ Toets van sphericiteit:
 - Huynh-Feldt &epsilon;: De Huynh-Feldt correctie.
 
 #### Contrasten
-Afwijking/Enkelvoudig/Verschil/Helmert/Repeated/Polynoom/Custom Contrast:
+Afwijking/Enkelvoudig/Verschil/Helmert/Herhaald/Polynoom/Handmatig Contrast:
 - Vergelijking: De niveaus van de onafhankelijke variabele die worden vergeleken. Voor de custom contrasten wordt het gewicht getoond voor elk niveau van de factor.
```
Comment (TimKDJ) [inline Resources/Help/analyses/anovarepeatedmeasures_nl.md]: And this one (custom)

**Comment by TimKDJ** [comment]
@boutinb please don't `git commit --amend` :( I can't see the changes you make

**Comment by boutinb** [comment]
@TimKDJ This time I have added a commit for the the new changes

**Review by TimKDJ — APPROVED** [review]

**Pull request merged**